amd_openvx_extensions/amd_nn/src/activation_layer.cpp

```diff
@@ -106,7 +106,7 @@ static vx_status VX_CALLBACK initializeActivationLayer(vx_node node, const vx_re
 {
     ActivationLayerLocalData * data = new ActivationLayerLocalData;
     memset(data, 0, sizeof(*data));
-    ERROR_CHECK_STATUS(createGraphHandle(node, &data->handle));
+    ERROR_CHECK_STATUS(createRPPHandle(node, &data->handle));
```

Comment:
why RPP handle has been changed here?
it is under amd_nn

@Akilesh2 we already made this change in performance PR and faced lot of issues
Please do not change files other than the RPP / relevelant files